index.bs

```diff
@@ -10141,87 +10093,67 @@ which allows construction of objects that
 implement the interface on which the
 [{{NamedConstructor}}] extended attributes appear.
 
-<div algorithm="named constructor">
-
-    It behaves as follows, assuming |arg|<sub>0..|n|−1</sub> is the list
-    of argument values passed to the constructor,
-    |id| is the identifier of the constructor specified in the
-    extended attribute [=takes a named argument list|named argument list=],
-    and |I| is the [=interface=] on which the
-    [{{NamedConstructor}}] extended attribute appears:
-
-    1.  Initialize |S| to the
-        [=effective overload set=]
-        for constructors with [=identifier=]
-        |id| on [=interface=]
-        |I| and with argument count |n|.
-    1.  Let &lt;|constructor|, |values|&gt; be the result of passing |S| and
-        |arg|<sub>0..|n|−1</sub> to the
-        [=overload resolution algorithm=].
-    1.  Let |R| be the result of performing the actions listed in the description of
-        |constructor| with |values| as the argument values.
-    1.  Return the result of [=converted to an ECMAScript value|converting=]
-        |R| to an ECMAScript [=interface type=] value |I|.
-</div>
-
-If the internal \[[Call]] method
-of the [=named constructor=]
-returns normally, then it must
-return an object that implements interface |I|.
-This object also must be
-associated with the ECMAScript global environment associated
-with the [=named constructor=].
+If the internal \[[Call]] method of the [=named constructor=] returns normally,
```

Comment:
Prexisting problem, but: this shouldn't be talking about the internal [[Call]] method. It should be talking about the steps that spec authors use to describe the named constructor ("the actions listed in the description of constructor" below).